rasa/telemetry.py

```diff
@@ -490,6 +491,10 @@ def _default_context_fields() -> Dict[Text, Any]:
             "cpu": multiprocessing.cpu_count(),
             "docker": _is_docker(),
         }
+        license_hash = plugin_manager().hook.get_license_hash()
+        print("LLLLLLLLLLLLLLL",license_hash)
```

Comment:
We should remove this print before merging 😄

Reply:
😶‍🌫️